docs/src/_includes/components/docs-index.html

```diff
@@ -10,16 +10,16 @@
 </li>
 {%- endmacro %}
 
-<nav class="docs-index" id="docs-index">
-    <button class="docs-index-toggle" aria-label="Index" id="js-docs-index-toggle" hidden>
+<nav class="docs-index" id="docs-index" >
+    <button class="docs-index-toggle" aria-label="Index" id="js-docs-index-toggle">
         Index
         <svg width="20" height="20" viewBox="20 20 60 60">
             <path id="ham-top" d="M30,37 L70,37 Z" stroke="currentColor"></path>
             <path id="ham-middle" d="M30,50 L70,50 Z" stroke="currentColor"></path>
             <path id="ham-bottom" d="M30,63 L70,63 Z" stroke="currentColor"></path>
         </svg>
     </button>
-    <ul class="docs-index__list" id="js-docs-index-list">
+    <ul class="docs-index__list" id="js-docs-index-list" data-open="false">
```

Comment:
This is the opposite, users without JavaScript should see this, so we can't hide it initially.